src/fluree/db/serde/avro.clj

```diff
@@ -1,387 +0,0 @@
-(ns fluree.db.serde.avro
```

Comment:
Will we need this for writing index blocks with the file conn?

Reply:
Added back in for now.